lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
```

Comment:
`Number.isNaN(value.valueOf())` seems like the better way to go to me, even if it does require a code comment.